### PR TITLE
Make code editor font family follow theme

### PR DESCRIPTION
**Pull request by KTibow**
## Proposed change

Lets themes change font family of code editor with `paper-font-common-code_-_font-family`.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml
frontend:
  themes:
    fontchange:
      paper-font-common-code_-_font-family: "Comic Sans MS, cursive, sans-serif"
```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-code-editor.ts [hunk 1]
```diff
@@ -97,6 +97,7 @@ export class HaCodeEditor extends UpdatingElement {
       .CodeMirror {
         height: var(--code-mirror-height, auto);
         direction: var(--code-mirror-direction, ltr);
+        font-family: var(--paper-font-common-code_-_font-family);
```
Comment (bramkragten) [inline src/components/ha-code-editor.ts:100]: We should not use paper variables

Comment (KTibow) [reply src/components/ha-code-editor.ts:100]: What instead? Create a new variable?

Comment (bramkragten) [reply src/components/ha-code-editor.ts:100]: Yes

Comment (KTibow) [reply src/components/ha-code-editor.ts:100]:
```suggestion
        font-family: var(--code-font-family, monospace);
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**